### PR TITLE
generate_package: Test version variable

### PR DESCRIPTION
**Pull request by chriscoyfish**
Fixes https://github.com/coala-analyzer/coala-bears/issues/597

### CODE REVIEW

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/690/commits/d4bd29f687723c4efd4266513d92107fc1361c55 - @Adrianzatreanu you definitely owe @chriscoyfish a bar of chocolate for fixing up your stuff ;)

Thank you so much @chriscoyfish !

File: tests/generate_packageTest.py [hunk 1]
```diff
@@ -99,6 +99,22 @@ def test_main(self):
...
+    def test_main_bear_version_prod(self):
+        fake_prod_version = '99.99.99'
+        with patch('bears.generate_package.VERSION', fake_prod_version):
+            main()
+        with open(self.CSS_BEAR_SETUP_PATH) as fl:
+            setup_py = fl.read()
+        self.assertIn(fake_prod_version, setup_py)
+        self.assertNotIn(VERSION, setup_py)
+
+    def test_main_bear_version_dev(self):
```
Comment (sils1297) [inline tests/generate_packageTest.py]: ah wait actually when we do a release, the version will be a production one for one commit which CI will be run against, I think we should rather patch this here to a dev version.

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/690/commits/d4bd29f687723c4efd4266513d92107fc1361c55 as per the pointed out thing

File: tests/generate_packageTest.py (hunk 1 above)
Comment (chriscoyfish) [inline tests/generate_packageTest.py]: Ack. Added dev patch as well.

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/690/commits/a6c61c0a24fd5a8718d862ef7cb17665745f0ccc

**Comment by sils1297** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.readthedocs.org/en/latest/Users/Tutorials/Git_Help.html) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://gitter.im/coala-analyzer/coala) if you need help!

**Comment by chriscoyfish** [comment]
Rebased. Thanks Lasse!

**Comment by sils1297** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/690/commits/71436f73ae64e0c173e38f4cfb952af796a75dca

**Comment by sils1297** [comment]
@rutor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
Awesome, thanks! :)

**Pull request merged**